database/README.md

```
@@ -0,0 +1,81 @@
+
+      Display Prisma version info
+      $ prisma version
+
+      Display Prisma debug info
+      $ prisma debug
+
+```
+
+
+
+# Prisma Migration
```

Markdownlint

Multiple top-level headings in the same document

(MD025, single-title, single-h1)